src/tools/kata-ctl/src/check.rs

```diff
@@ -332,6 +383,16 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_check_kvm_is_usable_generic() {
+        if !nix::unistd::Uid::effective().is_root() {
```

Comment:
We have a marcro [`skip_if_not_root`](https://github.com/kata-containers/kata-containers/blob/629a31ec6e9ee5744f53ed03a0eb9a2fa69f0b84/src/agent/src/network.rs#L90-L94) for running tests for root user.

Reply:
@liubin thanks, updated